test/utils/utils.go

```diff
@@ -247,3 +256,51 @@ func UncommentCode(filename, target, prefix string) error {
 	// nolint:gosec
 	return os.WriteFile(filename, out.Bytes(), 0644)
 }
+
+// LoadImage does not work when using podman, so LoadImageArchive is used instead
+// refer to https://github.com/kubernetes-sigs/kind/issues/2038
+func LoadContainerImageToKindCluster(ctx context.Context,
+	kindCluster *kind.Cluster,
+	dockerImage string,
+	containerTool string) (context.Context, error) {
```

Comment:
```suggestion
	containerTool string) error {
```

Reply:
It's a little weird to return the received context.

Reply:
True